cmd/juju/user/change_password_test.go

```diff
@@ -116,15 +121,60 @@ func (s *ChangePasswordCommandSuite) TestChangeOthersPassword(c *gc.C) {
 	s.assertAPICalls(c, "other", "sekrit")
 }
 
+func (s *ChangePasswordCommandSuite) TestResetPassword(c *gc.C) {
+	s.mockAPI.key = []byte("no cats or dragons")
+	context, _, err := s.run(c, "--reset")
+	c.Assert(err, jc.ErrorIsNil)
+	s.mockAPI.CheckCall(c, 0, "ResetPassword", "current-user")
+	c.Assert(cmdtesting.Stdout(context), gc.Matches, `
```

Comment:
well, i think the ourder of stderr/stdout depends on how stderr and stdout is configured on the client side?
The best I can do code-wise is either put both as ctx.Infof (which currently goes to stderr) or both on stdout explicitly.